R/report_htest_ttest.R

```diff
@@ -43,18 +43,21 @@
 
 # report_effectsize ---------------------
 
-.report_effectsize_ttest <- function(x, table, dot_args) {
+.report_effectsize_ttest <- function(x, table, dot_args, type, rules = "cohen1988") {
   args <- c(list(x), dot_args)
-  table <- do.call(effectsize::cohens_d, args)
+  table <- do.call(effectsize::effectsize, args)
```

Comment:
@mattansb I know you would much rather use the specific functions rather than the generic `effectsize::effectsize()`. Do you see any issues down the line due to this change?